### DISCUSSION TITLE
Seeking Advice on Managing IT for a Clinic

### DISCUSSION THREAD

**Discussion by 0Serenvale** [cockpit-project/cockpit#21047] [General]
Hi everyone,

I've recently joined a clinic as a temporary IT replacement while the current IT manager is on a month-long honeymoon. My background is primarily in web development and server optimization, so I'm relatively new to network management.

Here's the current setup at the clinic:

* **Server Location:** Basement
* **Network Hardware:**
   * Wi-Fi router
   * Two D-Link 24-port switches
      * One switch has internet access, the other does not
   * Each floor has two switches and a Wi-Fi router, with additional switches further within the building
* Connected Devices: \*\*\*\*  - Local network includes printers, medical equipment, and computers
   * Computers running various operating systems: Ubuntu, Windows 10, Windows 7, and Windows XP
   * New computers are added by connecting to the network and assigning a static IP
   * Around 10 computers use dynamic IPs and connect via Wi-Fi

**Challenges:**

* The network is quite complex and feels disorganized.
* The IT manager's computer has access to both subnet networks and uses Docker for managing clinic software.
* The IT manager is reluctant to share passwords for the server and his computer, concerned about His job.
* I have access to existing tools for network scanning but cannot install new software.

I'm considering resetting the root password for both the IT computer and server (running Ubuntu 16) using GRUB, and then potentially installing Cockpit to better manage the network. However, I'm cautious about making changes in a clinic environment, especially with such an outdated system.

Given the circumstances and my limited experience with large networks, I would greatly appreciate any advice, suggestions, or guidance on how to approach this situation. What steps should I take to ensure stability and security without risking disruptions to the clinic's operations?

Thank you in advance for your help!

Note:  **Understanding the Situation at the Clinic**

I wanted to provide some clarity on the situation at the clinic and explain why I'm considering certain actions.

The clinic's management is a family affair, with both the father and son making decisions. The current IT guy is a close friend of the son, and there's a belief that without him, the clinic's operations would collapse.

From what I've seen, there's a lot of work piling up, and the IT guy's approach seems to be quite lackadaisical. He acts as if he owns the place, arriving late around 9:30 AM, leaving for long breaks, and wrapping up his day early at 4:00 PM. Tasks rarely get completed promptly since he's often engaged in casual conversations.

His primary focus appears to be handling only critical issues, like rebooting systems when printers or networks act up, or checking cables.

Whenever a new IT person is hired, he withholds information from them, leading to inevitable conflicts. Ultimately, management sides with him, and the cycle continues.

Given these dynamics, I'm trying to navigate the situation carefully and would appreciate any advice on how to proceed effectively.

**Comment by martinpitt** [comment]
Hello @Rango-dz ! There is very little cockpit specific here -- it's mostly "general IT disaster waiting to happen". My main advice is to make sure that the ancient Ubuntu 16 (I take that as "16.04 LTS") is either not reachable from the network, or is still getting security updates through an extended commercial contract with Canonical. It sounds like "manage the network" hopefully boils down mostly to "check that it keeps running and don't make structural changes". In Cockpit you can monitor what's going on quite well (resource usage, logs, monitoring) and do the occasional "stop and restart service/interface" if necessary, but this should all be done for good reasons and with knowing the impact.

A good approach to achieve stability and security is to run redundant systems, supported OSes, and making sure that *everything* is automated -- i.e. encapsulate all the knowledge in Ansible (or puppet, etc.) config/deployment management, instead of having "snowflake" systems which exactly one person knows about. But this gets far beyond the scope of a cockpit discussion forum answer..
